db/views/reporting_facility_state_daily_dimensions_v01.sql

```diff
@@ -0,0 +1,224 @@
+WITH follow_up_blood_pressures AS (
```

Comment:
It makes sense to add the other code with this PR. But, then if I add those changes too to this PR, then this would be big to review. I can add it to a separate PR and then merge those two PR before merging to master.

Reply:
Could continue with this PR now since there's been a round of reviews. Also it's okay to let these matview definition PRs be a bit long since the actual diff is usually small.

Reply:
Hmm, okay